### PR TITLE
Replace inline javascript

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fix for #613 

cc @bummytime and @diegoreymendez can you guys have a look

### CODE REVIEW

File: Classes/WPEditorView.m [hunk 1]
```diff
@@ -1127,7 +1046,7 @@ - (BOOL)isVideoPressInfoRequestScheme:(NSString *)scheme
 
 - (BOOL)isSelectionStyleScheme:(NSString*)scheme
 {
-	static NSString* const kCallbackScheme = @"callback-video-fullscreen";
+	static NSString* const kCallbackScheme = @"callback-selection-style";
```
Comment (diegoreymendez) [inline Classes/WPEditorView.m]: Is this related to the issue being solved?

File: Classes/WPEditorView.m [hunk 2]
```diff
@@ -593,7 +512,7 @@ - (void)handleDOMLoadedCallback:(NSURL*)url
 {
     NSParameterAssert([url isKindOfClass:[NSURL class]]);
     
-    self.editorInteractionQueue = nil;
+//    self.editorInteractionQueue = nil;
```
Comment (diegoreymendez) [inline Classes/WPEditorView.m]: Is this commented on purpose?

File: Classes/WPEditorView.m (hunk 1 above)
Comment (SergioEstevao) [inline Classes/WPEditorView.m]: No this must got by mistake, it was another or that is merged in dev already

File: Classes/WPEditorView.m (hunk 2 above)
Comment (SergioEstevao) [inline Classes/WPEditorView.m]: It needs to be removed altogether

**Comment by bummytime** [comment]
Ahhh yes. This has been on our todo list forever. Nice job @SergioEstevao. 

It looks like you may need to `pod install` the example project (the project file was dirty for me after I did so). Beyond that, I tested things and everything still seems to work. So a :shipit: from me.
